public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchRequest.ts

```diff
@@ -50,13 +50,18 @@ export abstract class CloudWatchRequest {
         return { ...result, [key]: null };
       }
 
-      const newValues = this.expandVariableToArray(value, scopedVars);
+      const newValues = this.expandVariableToArrayIfTemplated(value, scopedVars);
       return { ...result, [key]: newValues };
     }, {});
   }
 
   // get the value for a given template variable
-  expandVariableToArray(value: string, scopedVars: ScopedVars): string[] {
+  expandVariableToArrayIfTemplated(value: string, scopedVars: ScopedVars): string | string[] {
```

Comment:
You don't need to change the name of this method - if it's a variable it's always templated.

Reply:
I added the `IfTemplated` because I figured changing the method signature to return `string | string[]` means that there should be some indication that it won't always strictly be an Array (`string[]`), but then again I guess the typing already says that.